src/core/utils/api-client/notifications.ts

```diff
@@ -59,6 +62,15 @@ export class NotificationsApiClient {
                 return response.result.notifications;
             }
         } catch (err) {
+            SentryAddBreadcrumb({
+                message: JSON.stringify({
+                    data: {
+                        blockchainNetworks
+                    },
+                    err
+                })
+            });
+
             SentryCaptureException(new Error(JSON.stringify(err)));
```

Comment:
You need to replace all these `SentryCaptureException(new Error(JSON.stringify(err)));` with specific error messages